db/reqlog.c

```diff
@@ -3068,6 +3071,11 @@ inline void reqlog_set_error(struct reqlogger *logger, const char *error,
     logger->error_code = error_code;
 }
 
+inline void reqlog_set_api_type(struct reqlogger *logger, const char *api_type)
+{
+    if (logger) logger->api_type = strdup(api_type);
```

Comment:
I think this needs to be freed. But just pointing it to the string literal is probably cleaner.

Reply:
Thanks for catching this :sweat_smile: yeah I agree that the string literal is cleaner.